caab-api/open-api-specification.yml

```diff
@@ -169,6 +169,38 @@ paths:
           description: 'Not found'
         '500':
           description: 'Internal server error'
+    delete:
+      tags:
+        - applications
+      summary: 'Remove an application'
+      operationId: 'removeApplication'
+      parameters:
+        - name: 'id'
+          in: 'path'
+          required: true
+          schema:
+            type: 'integer'
+            format: 'int64'
+            example: '1234567890'
+        - name: 'Caab-User-Login-Id'
```

Comment:
Is this not a referenced parameter somewhere?

Reply:
It wasn't, but it is now. That and a shared 'id' parameter now added and referenced everywhere.